sdk/resourcemanager/Azure.ResourceManager.Core/tests/Scenario/ResourceManagerTestBase.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.TestFramework;
+
+namespace Azure.ResourceManager.Core.Tests
```

Comment:
We should update the template to reflect this new folder structure and new files. could you add an ado

Reply:
https://dev.azure.com/azure-mgmt-ex/DotNET%20Management%20SDK/_workitems/edit/5633